CHANGELOG.md

```diff
@@ -142,6 +142,9 @@ Adding a new version? You'll need three changes:
 
 ### Added
 
+- Added support for expression-based Kong routes for `TLSRoute`. This requires the `ExpressionRoutes`
+  feature gate and a Kong 3.4.2 install with `KONG_ROUTER_FLAVOR=expressions` set in the environment.
```

Comment:
Maybe we do not need to mention version of Kong here since KIC 3.0 requires Kong >= `3.4.1` ? @programmer04

Reply:
@randmonkey So I remove the version requirement here?

Reply:
Yes, I think 
```suggestion
  feature gate and a Kong installed with `KONG_ROUTER_FLAVOR=expressions` set in the environment.
```
like this is OK. And please limit a line to contain at most 80 characters.